src/Transformers/FamilyTransformer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Choredo\Transformers;
+
+use Assert\Assertion;
+use Choredo\Entities;
+use League\Fractal\TransformerAbstract;
+use const Choredo\DAYS_OF_WEEK;
+
+class FamilyTransformer extends TransformerAbstract
+{
+    public function transform(Entities\Family $family)
+    {
+        return [
+            'id'         => $family->getId()->toString(),
+            'name'                => $family->getName(),
+            'paymentStrategy'     => $family->getPaymentStrategy(),
+            'completionThreshold' => $family->getCompletionThreshold(),
+            'weekStartDay'        => $this->transformWeekStartDay($family->getWeekStartDay()),
+        ];
+    }
+
+    private function transformWeekStartDay(int $weekStartDay): string
+    {
+        Assertion::between($weekStartDay, 0, count(DAYS_OF_WEEK) - 1);
```

Comment:
Is this the correct place to be doing assertions? Shouldn't that logic be handled elsewhere? By the time it gets to the presentation stage, if there's a logic problem it's big one.

Reply:
If you are going to make assertions, I would add a test for the fail case.